source/fuzz/fuzzer_context.h

```diff
@@ -327,6 +333,17 @@ class FuzzerContext {
   uint32_t GetRandomIndexForCompositeInsert(uint32_t number_of_components) {
     return random_generator_->RandomUint32(number_of_components);
   }
+  int64_t GetRandomValueForStepConstantInLoop() {
+    // INT32_MAX is the maximum value that can be used with RandomUint32. Each
+    // call to RandomUint32 will return a 31-bit number (the 32-th bit - the
+    // sign bit - is always 0). By appending them together, we get a number
+    // between 0 and 2^62-1 (included).
+    uint64_t result = random_generator_->RandomUint32(INT32_MAX);
+    result <<= 31u;
+    result += random_generator_->RandomUint32(INT32_MAX);
+    // We choose the sign, so the number returned is in the range (-2^62, 2^62).
+    return random_generator_->RandomBool() ? result : (0 - result);
```

Comment:
Am I right that what you'd really like is a function, RandomInt64, that could be invoked on random_generator_?  If so then I think it would be better to simply add that function rather than resort to pasting together two integers.

Reply:
Indeed. It is now used after being introduced by #3809 .